content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -95,7 +98,8 @@ deployment.apps/nginx-deployment created
 
 ### Required fields
 
-In the `.yaml` file for the Kubernetes object you want to create, you'll need to set values for the following fields:
+In the manifest (YAML or JSON file) for the Kubernetes object you want to create, you'll need to set values for
+the following fields:
 
 * `apiVersion` - Which version of the Kubernetes API you're using to create this object
```

Comment:
```suggestion
* `apiVersion` - The version of the Kubernetes API you're using to create this object.
* `kind` - The kind of object you want to create.
* `metadata` - Data that helps uniquely identify the object, including a `name` string, `UID`, and optional `namespace`.
* `spec` - The desired state for the object.
```
Just to be consistent , also we can make metadata description to 
```
* `metadata` - What attributes or data you want to provide that helps uniquely identify the object, including a `name` string, `UID`, and optional `namespace`.
```

Reply:
Watch out for this gotcha @Ritikaa96 

> When a PR updates an existing page, you should focus on reviewing the parts of the page that are being updated. That changed content should be reviewed for technical and editorial correctness. If you find errors on the page that don't directly relate to what the PR author is attempting to address, then it should be treated as a separate issue (check that there isn't an existing issue about this first).

(from https://kubernetes.io/docs/contribute/review/reviewing-prs/#reviewing-checklist)

Also, though, I disagree with the suggestion; for me the original text works better and reads like idiomatic UK English. It might not be idiomatic US English; I'm not sure.

Reply:
Ah.. Noted. thanks for mentioning this @sftim